openproblems/tasks/dimensionality_reduction/README.md

```diff
@@ -41,10 +41,17 @@ data for visualization and interpretation.
 
 ## API
 
-**Datasets** should provide un-normalized raw counts in `adata.X`.
+WARNING: other than most tasks, `adata.X` should contain log-normalized data.
+   This is the case as we are computing ground truth metrics on normalized data,
+   which means methods which use this same normalization are likely to score more
+   highly on these metrics.
```

Comment:
Should we mention the normalisation function here?

Reply:
Yep!